module2/src/com/epam/onedimensionalarrays/ArrayWithoutMinValue.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.onedimensionalarrays;
+
+public class ArrayWithoutMinValue {
+    public static int[] findAndDeleteMinValue(int[] array) {
+        int minCount = 0;
```

Comment:
I want to make you change this variable, e.g.
```suggestion
        int counterOfMinValues = 0;
```